pkg/controller.v1/pytorch/job.go

```diff
@@ -216,3 +216,8 @@ func getTotalFailedReplicas(job *pyv1.PyTorchJob) int32 {
 	}
 	return totalFailedReplicas
 }
+
+func getPriorityClassName(job *pyv1.PyTorchJob) string {
+	priorityClassName := *(job.Spec.PriorityClassName)
```

Comment:
There may be an runtime error: invalid memory address or nil pointer dereference

Reply:
I need to remove the bracket

Reply:
right？ @gaocegege

Reply:
No I think you need to check if it is nil.

Reply:
I got it.